{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/settings/models.py

```diff
@@ -53,10 +61,12 @@ class Meta:
     def __str__(self):
         return self.name
 
+    @cached_property
     def value(self):
         return {
             'string': self.string,
             'text': linebreaksbr(mark_safe(self.text)),
+            'html': mark_safe(self.html),
```

Comment:
It doesn't look like you've added 'html' as a choice in the `type` field.

Reply:
Thats because I'm an idiot. Fixed! :)